tendrl/node_agent/monitoring/collectd/collectors/gluster/heavy_weight/tendrl_gluster_heal_info.py

```diff
@@ -1,15 +1,12 @@
 import collectd
-import socket
-import sys
 import traceback
+import threading
+
 
-sys.path.append('/usr/lib64/collectd/gluster')
 import utils as tendrl_glusterfs_utils
-sys.path.remove('/usr/lib64/collectd/gluster')
 
 
-CLUSTER_TOPOLOGY = {}
-CONFIG = {}
+ret_val = {}
```

Comment:
Do we really need ret_val as global?

Reply:
Currently no. But whenever gluster optimizes their transaction model, we can deploy python threads to get info per volume(for better scalability which is now not possible because of the "Another transaction in progress" limitation if we try to parallelize it)
All data are for different keys in ret_val dict and so it is thread-safe writes even if multiple threads are writing to this global variable.
This code is kept ready for parallelization by just uncommenting the commented lines in https://github.com/Tendrl/node-agent/pull/599/files#diff-eb9b46f14f5541bc2ebc3c769da9efb6R134 to 145
and commenting out https://github.com/Tendrl/node-agent/pull/599/files#diff-eb9b46f14f5541bc2ebc3c769da9efb6R133